### ISSUE TITLE
devours cpu when loading results

### ISSUE THREAD

**Issue by FlashlightET** [stevenwaterman/musetree#101]
![image](https://user-images.githubusercontent.com/29938499/147272793-47b29032-b8c3-4c27-9136-cc9ee4696f69.png)

when loading a lot of results it uses quite a bit of cpu (maybe bit is an understatement)

what causes this why should loading this data take so much cpu on my end

**Comment by stevenwaterman** [comment]
Yep, it's very heavy. Musetree uses the MuseNet API to generate the audio, which returns it as a midi. To play it back, we have to synthesise it into actual audio. We do that as fast as possible, which means it uses as much CPU as it can.
